charts/sonarr/values.yaml

```diff
@@ -1,153 +1,11 @@
 # Default values for sonarr.
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
 
-image:
-  repository: linuxserver/sonarr
-  tag: 3.0.3.911-ls39
-  pullPolicy: IfNotPresent
-
-# upgrade strategy type (e.g. Recreate or RollingUpdate)
-strategyType: Recreate
-
-# Probes configuration
-probes:
-  liveness:
-    initialDelaySeconds: 60
-    failureThreshold: 5
-    timeoutSeconds: 10
-  readiness:
-    initialDelaySeconds: 60
-    failureThreshold: 5
-    timeoutSeconds: 10
-
-# Prometheus Metrics
-exportarr:
-  enabled: false
+sonarr:
   image:
-    repository: onedr0p/exportarr
-    tag: v0.3.0
+    organization: itscontained
```

Comment:
Why do we switch from linuxserver/sonarr to itscontained/sonarr here? The linuxserver images are multi-arch and much wider used.

Reply:
This is a subjective choice and i can revert the default to linuxserver if you prefer. The itscontained containers have s6 removed and more closely align with docker's best practices and intent for containerization.